jwst/background/background_sub.py

```diff
@@ -112,9 +120,9 @@ def get_subset_array(self, other):
         dq_cutout = other.dq[idx]
 
         # Put them into the right place in the original image array shape
-        data_overlap[:data_cutout.shape[0], :data_cutout.shape[1]] = copy.deepcopy(data_cutout)
-        err_overlap[:data_cutout.shape[0], :data_cutout.shape[1]] = copy.deepcopy(err_cutout)
-        dq_overlap[:data_cutout.shape[0], :data_cutout.shape[1]] = copy.deepcopy(dq_cutout)
+        data_overlap[..., :data_cutout.shape[-2], :data_cutout.shape[-1]] = copy.deepcopy(data_cutout)
```

Comment:
I know these were already here before this PR, so it isn't strictly related, but why is the deepcopy necessary here?

Reply:
I don't remember exactly, but I presume I put these in because weird stuff was happening if I just assigned that slice to the array itself

Reply:
I probably put these here -- don't remember exactly why, but I presume I was worried about weird behaviour with arrays if a deepcopy wasn't used